google/cloud/bigtable/internal/partial_result_set_source.cc

```diff
@@ -17,13 +17,29 @@
 #include "google/cloud/internal/absl_str_cat_quiet.h"
 #include "google/cloud/internal/make_status.h"
 #include "google/cloud/log.h"
+#include "absl/strings/cord.h"
 #include "absl/types/optional.h"
 
 namespace google {
 namespace cloud {
 namespace bigtable_internal {
 
 GOOGLE_CLOUD_CPP_INLINE_NAMESPACE_BEGIN
+namespace {
+// Some Bigtable proto fields use Cord internally and string externally.
+template <typename T, typename std::enable_if<
+                          std::is_same<T, std::string>::value>::type* = nullptr>
+std::string AsString(T const& s) {
+  return s;
+}
+
+template <typename T, typename std::enable_if<
+                          std::is_same<T, absl::Cord>::value>::type* = nullptr>
+std::string AsString(T const& s) {
+  // Add the missing include: #include "absl/strings/cord.h"
```

Comment:
We can remote this comment and the one on the return statement.

Reply:
Removed debugging comments, thank you for catching this!